Source/WebKit/UIProcess/Extensions/Cocoa/_WKWebExtensionDeclarativeNetRequestSQLiteStore.mm

```diff
@@ -264,7 +264,7 @@ - (NSURL *)_databaseURL
 
     ASSERT([_storageType isEqualToString:@"dynamic"]);
 
-    NSString *databaseName = @"DynamicRules-DeclarativeNetRequest.db";
+    NSString *databaseName = @"DeclarativeNetRequestDynamicRules.db";
```

Comment:
I don't understand what this change does.  Does it lose existing data?  Are the compiled content rule lists put into an sqlite database?

Reply:
This is a separate DB for rules the extension can change dynamically. I'll split this out like you suggested above.